src/libxrpl/protocol/LedgerFormats.cpp

```diff
@@ -364,6 +364,36 @@ LedgerFormats::LedgerFormats()
             {sfPreviousTxnLgrSeq,   soeREQUIRED}
         },
         commonFields);
+		
+	add(jss::MPTokenIssuance,
+        ltMPTOKEN_ISSUANCE,
+        {
+            {sfIssuer,               soeREQUIRED},
+            {sfSequence,             soeREQUIRED},
+            {sfTransferFee,          soeDEFAULT},
+            {sfOwnerNode,            soeREQUIRED},
+            {sfAssetScale,           soeDEFAULT},
+            {sfMaximumAmount,        soeOPTIONAL},
+            {sfOutstandingAmount,    soeREQUIRED},
```

Comment:
It might be worth updating `sfOutstadingAmount` here and in the spec to be `soeDEFAULT`. The reasoning is that any new `MPTokenIssuances` and any that have been fully redeemed and intended to be deleted will have a 0 amount, which can be represented with the default. That's _probably_ not going to be very common, so isn't required. What do you think?

Reply:
> Though i would say having `OutstandingAmount` set to 0 won't have as much of a space impact as `MPTAmount` in `MPToken`

Yeah. That's a good point. I'm fine with leaving it as-is.